numpy/f2py/rules.py

```diff
@@ -114,7 +114,7 @@
  * f2py is a Fortran to Python Interface Generator (FPIG), Second Edition,
  * written by Pearu Peterson <pearu@cens.ioc.ee>.
  * See http://cens.ioc.ee/projects/f2py2e/
- * Generation date: """ + time.asctime(time.localtime(time.time())) + """
+ * Generation date: """ + time.asctime(time.gmtime(int(os.environ.get('SOURCE_DATE_EPOCH', time.time())))) + """
```

Comment:
If you are going to keep this, could you break it so that the line length is < 80 chars?